src/start_server.ts

```diff
@@ -150,7 +155,40 @@ export function getApp(options: ServerOptions): express.Express {
 
   app.use('/components/', polyserve);
 
+  if (options.proxy) {
+    if (options.proxy.path.startsWith('components')) {
+      console.error('proxy path can not start with components.');
+      return;
+    }
+
+    let escapedPath = '';
+    let escaped = false;
+    for (const char of options.proxy.path) {
+      escaped = (char === '\\') ? !escaped : false;
+      if (!escaped && (char === '*' || char === '+' || char === '?')) {
+        escapedPath += '\\\\';
```

Comment:
Why are you writing two backslashes here? This would cause the following character to still be unescaped, unless I'm missing something.